doc/thirdpartypackages/index.rst

```diff
@@ -1,179 +1,147 @@
-*********************
+********************
+Third party packages
+********************
 
 Several external packages that extend or build on Matplotlib functionality
-exist. Below we list a number of these. Note that they are each
-maintained and distributed separately from Matplotlib, and will need
-to be installed individually.
+exist. Below we list a number of these. Note that they are each maintained
+and distributed separately from Matplotlib, and will need to be installed
+individually.
 
-Please submit an issue or pull request
-on Github if you have created a package that you would like to have included.
-We are also happy to host third party packages within the `Matplotlib Github
-Organization <https://github.com/matplotlib>`_.
-
-.. _hl_plotting:
+Please submit an issue or pull request on Github if you have created
+a package that you would like to have included.  We are also happy to
+host third party packages within the `Matplotlib Github Organization
+<https://github.com/matplotlib>`_.
 
 High-Level Plotting
 *******************
 
-Several projects provide higher-level interfaces for creating
-matplotlib plots.
-
-.. _toolkit_seaborn:
-
-seaborn
-=======
-
-`seaborn <http://seaborn.pydata.org/>`_ is a high level interface for drawing
-statistical graphics with matplotlib. It aims to make visualization a central
-part of exploring and understanding complex datasets.
-
-.. image:: /_static/seaborn.png
-    :height: 157px
-
-.. _toolkit_ggplot:
+Several projects provide higher-level interfaces for creating Matplotlib plots.
 
 ggplot
 ======
-
 `ggplot <https://github.com/yhat/ggplot>`_ is a port of the R ggplot2 package
-to python based on matplotlib.
+to python based on Matplotlib.
 
 .. image:: /_static/ggplot.png
     :height: 195px
 
-.. _toolkit_holoviews:
-
 holoviews
 =========
-
 `holoviews <http://holoviews.org>`_ makes it easier to visualize data
-interactively, especially in a `Jupyter notebook
-<http://jupyter.org>`_, by providing a set of declarative
-plotting objects that store your data and associated metadata.  Your
-data is then immediately visualizable alongside or overlaid with other
-data, either statically or with automatically provided widgets for
-parameter exploration.
+interactively, especially in a `Jupyter notebook <http://jupyter.org>`_, by
+providing a set of declarative plotting objects that store your data and
+associated metadata.  Your data is then immediately visualizable alongside or
+overlaid with other data, either statically or with automatically provided
+widgets for parameter exploration.
 
 .. image:: /_static/holoviews.png
     :height: 354px
 
+seaborn
+=======
+`seaborn <http://seaborn.pydata.org/>`_ is a high level interface for drawing
+statistical graphics with Matplotlib. It aims to make visualization a central
+part of exploring and understanding complex datasets.
 
-.. _toolkits-mapping:
+.. image:: /_static/seaborn.png
+    :height: 157px
 
 Mapping Toolkits
 ****************
```

Comment:
Yeah yeah... I do think these should get higher priority though as they support the only fundamental chart types (geospatial seems to get lumped in as basic on most every survey I've seen of viz types) matplotlib really doesn't support. .

Then I'm thinking group all the math ones together: seaborn, pyUbset, mpl-prob, then group the interactive, or the declarative, and then *shrugs*